hinks.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+
+"""
+Instructions
+
+- Functions intended to run for each scan
+  need to be named run_xxxxx
+
+- Do not modify the parameters of the run_xxx function
+  unless you know what you are doing
+  see optional parameters:
+  https://github.com/tangible-landscape/grass-tangible-landscape/wiki/Running-analyses-and-developing-workflows#python-workflows
+
+- All gs.run_command/read_command/write_command/parse_command
+  need to be passed env parameter (..., env=env)
+"""
+
+import grass.script as gs
+
+def run_ponds(scanned_elev, env, **kwargs):
+    repeat = 20
+    input_dem = scanned_elev
+    output = "tmp_filldir"
+    for i in range(repeat):
+        gs.run_command('r.fill.dir', input=input_dem, output=output, direction="tmp_dir", env=env)
+        input_dem = output
+    # filter depression deeper than 0.1 m to
+    gs.mapcalc('{new} = if({out} - {scan} > 0.1, {out} - {scan}, null())'.format(new='ponds', out=output,
+                                                                                 scan=scanned_elev), env=env)
+    gs.write_command('r.colors', map='ponds', rules='-', stdin='0% aqua\n100% blue', env=env)
+# this part is for testing without TL
+
+def run_waterflow(scanned_elev, env, **kwargs):
```

Comment:
There should be two empty lines in between functions according to the PEP8 code style standard for Python, so that's what the Flake8 linting tool wants you to do here and before `def main():` (the first error is the same as from the import linter). The relevant output from Super-Linter (which includes Flake8) is here:

```
2021-02-11 15:54:01 [ERROR]   Found errors in [flake8] linter!
2021-02-11 15:54:01 [ERROR]   Error code: 1. Command output:
------
/github/workspace/hinks.py:20:1: E302 expected 2 blank lines, found 1
/github/workspace/hinks.py:33:1: E302 expected 2 blank lines, found 1
/github/workspace/hinks.py:39:1: E302 expected 2 blank lines, found 1
```